libavformat/hls.c

```diff
@@ -706,6 +707,7 @@ static int parse_playlist(HLSContext *c, const char *url,
     struct segment **prev_segments = NULL;
     int prev_n_segments = 0;
     int prev_start_seq_no = -1;
+    double discont_program_date_time = -1;
```

Comment:
What does `discont` mean?

Reply:
`#EXT-X-PROGRAM-DATE-TIME` can show up at the beginning of the playlist, or right after a `#EXT-X-DISCONTINUITY`, to hint absolute timestamp; hence, the name `discont_program_date_time`.